Devscord.DiscordFramework/Middlewares/Contexts/MessageContext.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Devscord.DiscordFramework.Framework.Architecture.Middlewares;
+
+namespace Devscord.DiscordFramework.Middlewares.Contexts
+{
+    public class MessageContext : IDiscordContext
+    {
+        public string Content { get; }
+        public string AuthorName { get; }
```

Comment:
author to powinien być UserContext

Reply:
To nie miałoby sensu, przecież użytkownik nie ma swojego autora ;) To wiadomość ma autora.

Reply:
czekaj tobie chodziło, że zamiast przekazywać informacje o autorze tak jak teraz, to ma być tak, że zostanie przekazany UserContext, który opisuje autora i to zostaje przekazane do MessageContext?

Reply:
Potwierdzam 

Żeby nie było długiej listy propertiesów jak można to skrócić 
Co prawda teraz nie ma wielu, ale pewne informacje można połączyć w pary

Reply:
Chodzi o czytelność z punktu osoby przeglądającej końcowy obiekt